example/gluon/style_transfer/dataset/download_dataset.sh

```diff
@@ -0,0 +1,7 @@
+cd dataset
+wget http://msvocds.blob.core.windows.net/coco2014/train2014.zip
+wget http://msvocds.blob.core.windows.net/coco2014/val2014.zip
```

Comment:
Does this example really require full 12GB of coco? Would it be possible to trim it down? Use mx.test_utils.download instead for portability.